hc/api/transports.py

```diff
@@ -244,3 +251,15 @@ def notify(self, check):
         }
 
         return self.post(self.channel.value, payload)
+
+def custom_message(check):
```

Comment:
Always use OOP. If this must be a method that is used in only the `Telegram` class, why is not in that class? Also use a more descriptive and action like variable name for the method. An example of this would be `format_message` or something similar